Meter.c

```diff
@@ -50,8 +53,10 @@ Meter* Meter_new(const struct ProcessList_* pl, unsigned int param, const MeterC
    return this;
 }
 
+static const char* const Meter_prefixes = "KMGTPEZY";
+
 int Meter_humanUnit(char* buffer, unsigned long int value, size_t size) {
-   const char* prefix = "KMGTPEZY";
+   const char* prefix = Meter_prefixes;
    unsigned long int powi = 1;
    unsigned int powj = 1, precision = 2;
```

Comment:
```suggestion
   double powi = 1; // Scaling
   double powj = 1; // Precision checking
   int precision = 2;
```

Otherwise you overflow `powi` for values larger then EiB … Scaling with 1024. will always be precise, even with IEEE 754.

Also: printf and alike want the precision argument to be provided as `int`

Reply:
Will address this issue in #1321.